<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>11/3/0</td>
</tr><tr>
<td>Started on:</td><td>Mon Dec 16 20:02:40 GMT 2019</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (106 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test1()'><b>test1</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at com.testNG.AssertionInTestNG.test1(AssertionInTestNG.java:17)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace487694075", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace487694075'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.testNG.AssertionInTestNG.test1(AssertionInTestNG.java:17)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
</pre></div></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test3()'><b>test3</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td><div><pre>java.lang.AssertionError: expected [Test1] but found [Test]
	at com.testNG.AssertionInTestNG.test3(AssertionInTestNG.java:29)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace611572016", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace611572016'><pre>java.lang.AssertionError: expected [Test1] but found [Test]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.testNG.AssertionInTestNG.test3(AssertionInTestNG.java:29)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
</pre></div></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test5()'><b>test5</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td><div><pre>java.lang.AssertionError:  Test Case Failed expected [true] but found [false]
	at com.testNG.AssertionInTestNG.test5(AssertionInTestNG.java:35)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace77269878", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace77269878'><pre>java.lang.AssertionError:  Test Case Failed expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.testNG.AssertionInTestNG.test5(AssertionInTestNG.java:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
</pre></div></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.testNG.MyTestClass1.myTestMethod1()'><b>myTestMethod1</b><br>Test class: com.testNG.MyTestClass1</td>
<td></td>
<td>0</td>
<td>com.testNG.MyTestClass1@25a65b77</td></tr>
<tr>
<td title='com.testNG.MyTestClass1.myTestMethod2()'><b>myTestMethod2</b><br>Test class: com.testNG.MyTestClass1</td>
<td></td>
<td>0</td>
<td>com.testNG.MyTestClass1@25a65b77</td></tr>
<tr>
<td title='com.testNG.MyTestClass2.myTestMethod3()'><b>myTestMethod3</b><br>Test class: com.testNG.MyTestClass2</td>
<td></td>
<td>0</td>
<td>com.testNG.MyTestClass2@12d3a4e9</td></tr>
<tr>
<td title='com.testNG.MyTestClass2.myTestMethod4()'><b>myTestMethod4</b><br>Test class: com.testNG.MyTestClass2</td>
<td></td>
<td>0</td>
<td>com.testNG.MyTestClass2@12d3a4e9</td></tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test()'><b>test</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
<tr>
<td title='com.testNG.BasicAnnotationsInTestNG.test10()'><b>test10</b><br>Test class: com.testNG.BasicAnnotationsInTestNG</td>
<td></td>
<td>0</td>
<td>com.testNG.BasicAnnotationsInTestNG@2c34f934</td></tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test2()'><b>test2</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
<tr>
<td title='com.testNG.BasicAnnotationsInTestNG.test5()'><b>test5</b><br>Test class: com.testNG.BasicAnnotationsInTestNG</td>
<td></td>
<td>0</td>
<td>com.testNG.BasicAnnotationsInTestNG@2c34f934</td></tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test6()'><b>test6</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test7()'><b>test7</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
<tr>
<td title='com.testNG.AssertionInTestNG.test8()'><b>test8</b><br>Test class: com.testNG.AssertionInTestNG</td>
<td></td>
<td>0</td>
<td>com.testNG.AssertionInTestNG@240237d2</td></tr>
</table><p>
</body>
</html>